content/graphing/dashboards/screenboard.md

```diff
@@ -60,15 +61,16 @@ This feature can be enabled by following these simple steps:
 This Chrome plug-in helps you change the Color Scheme for your browser (Contrast, Reverse Colors, Image colors, Luminance) [Deluminate](https://chrome.google.com/webstore/detail/deluminate/iebboopaeangfpceklajfohhbpkkfiaa?hl=en-US).
 
 Simple to setup and easily activated/unactivated, Deluminate is a good fit for Datadog Screenboards.
-{{< img src="developers/faq/deluminate.png" alt="deluminate" responsive="true" popup="true">}}
+
+{{< img src="developers/faq/deluminate.png" alt="deluminate" responsive="true" popup="true" style="width:50%;">}}
 
 ### Stylebot (Chrome)
 
 Stylish offers more possibilities to customize the css of any web page.
```

Comment:
I realize you didn't add this, but the title is stylebot and the text says stylish

Reply:
but then again, why is this even in here. seems inappropriate to have instructions for a third party software in our docs.